### ISSUE TITLE
Multisite Support

### ISSUE THREAD

**Issue by arielk** [KingYes/wordpress-aryo-activity-log#45]

**Comment by ericandrewlewis** [comment]
I don't think you need a single place to track all the network activity in one place. This will not scale well. I'm moving a client off of the Stream plugin because it uses one activity log table for the entire site, onto this plugin.

Yes, Network-level activity would be nice (e.g. create user, create site, network activate plugin, etc.)

**Comment by eangulus** [comment]
Why not give the users a choice. Either individually activate per subsite and therefore track only that site. Or Network activate and we get what we have now with subsite tracking, but also get a Network Wide track under network admin. Also if network activated, give an option to turn off access to the subsite admins.

**Comment by colomet** [comment]
In our situation, we allow to our authors to write in different sites of the multisite. To track it by opening the different sites, is imposible. One centralice place would allow to know if they as jumping from site to site or just not working.

**Comment by colomet** [comment]
@KingYes do you have any plain for integrating such functionality in your plugin in a short-medium time frame?
